docs/development/development-tools.md

```diff
@@ -0,0 +1,119 @@
+# Development Tools
+
+This document provides an overview of two different but related development approaches -
+devcontainers (via Visual Studio Code), and command line tools via a Docker container.
```

Comment:
this is great! I've also been using vscode within running docker containers using VSCode's Docker extension. We should really have some sort of skill-share thing with anyone who does development for KBase as most people don't take advantage of what VSCode can do because it's a bit of a PITA to set up.